templates/cpu-compatibility/base_cpu-compatibility.html

```diff
@@ -0,0 +1,7 @@
+{% extends "templates/base.html" %}
+
+{% block meta_copydoc %}{% endblock meta_copydoc %}
```

Comment:
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/1T7BXK7-IjN9Rq_5C4PYTVGkuv5mxhIX7OeFTsY6pTKs/edit#{% endblock meta_copydoc %}
```
Here we usually include the brief, it isn't so important for a single page like this but it can still be useful in the future :)